xml/System.Numerics/Complex.xml

```diff
@@ -2851,6 +2968,39 @@ This API is not CLS-compliant.</summary>
         <altmember cref="M:System.Numerics.Complex.op_Equality(System.Numerics.Complex,System.Numerics.Complex)" />
       </Docs>
     </Member>
+    <MemberGroup MemberName="op_Multiply">
+      <Docs>
+        <summary>Multiplies a specified number by another specified number, where at least one of them is a complex number, and the other could be a double-precision real number.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The <xref:System.Numerics.Complex.op_Multiply%2A> operators allow performing multiplication operations that involve complex numbers. They enable code such as the following:
+
+[!code-csharp[System.Numerics.Complex.Multiply#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.numerics.complex.multiply/cs/multiply2.cs#2)]
+[!code-vb[System.Numerics.Complex.Multiply#2](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.numerics.complex.multiply/vb/multiply2.vb#2)]
+
+The multiplication of a complex number, a + bi, and a second complex number, c + di, takes the following form:
```

Comment:
The same comment applies here:
- Multiplication with the individual overloads should be removed from here, since it's repeated for the individual overload.
- The overflow and custom operators sentences can remain here and be removed in the individual overloads.